sig-scalability/throughput_burts_slo.md

```diff
@@ -0,0 +1,23 @@
+# SLO: Kubernetes cluster of size at least X is able to start Y Pods in Z minutes
```

Comment:
I vote for creating a dedicated directory for those documents, probably sig-scalability/slo/ (or sth like that).